lib/milestone-db/memory.js

```diff
@@ -22,12 +23,17 @@ function MemoryMilestoneDB(options) {
 MemoryMilestoneDB.prototype = Object.create(MilestoneDB.prototype);
 
 MemoryMilestoneDB.prototype.getMilestoneSnapshot = function (collection, id, version, callback) {
+  if (!callback) callback = function () {};
+  if (!collection) return process.nextTick(callback, new ShareDBError(4001, 'Invalid collection'));
```

Comment:
I'd go with "Missing collection" and "Missing ID" here to be more precise about the error, and same in `saveMilestoneSnapshot`.

Reply:
Fine by me. These errors are sort of "dummy" errors anyway, given that nobody should really be using this implementation (hence just using the `4001` code, and not wanting to invent new error codes)